docs/user/versions.rst

```diff
@@ -160,3 +92,42 @@ for example ``https://pip.readthedocs.io/``,
 they will be redirected to the **Default version**.
 This defaults to **latest**,
 but could also point to your latest released version.
+
+How we envision versions working
+--------------------------------
+
+In the normal case,
+the ``latest`` version will always point to the most up to date development code.
+If you develop on a branch that is different than the default for your VCS,
+you should set the **Default Branch** to that branch.
+
+You should push a **tag** for each version of your project.
+These tags should be numbered in a way that is consistent with semantic versioning.
+This will map to your ``stable`` branch by default.
+
+.. note::
+    We in fact are parsing your tag names against the rules given by
+    `PEP 440`_. This spec allows "normal" version numbers like ``1.4.2`` as
+    well as pre-releases. An alpha version or a release candidate are examples
+    of pre-releases and they look like this: ``2.0a1``.
+
+    We only consider non pre-releases for the ``stable`` version of your
+    documentation.
+
+If you have documentation changes on a **long-lived branch**,
+you can build those too.
+This will allow you to see how the new docs will be built in this branch of the code.
+Generally you won't have more than 1 active branch over a long period of time.
+The main exception here would be **release branches**,
+which are branches that are maintained over time for a specific release number.
+
+.. _PEP 440: https://www.python.org/dev/peps/pep-0440/
+
+Logging out
+-----------
+
+When you log in to a documentation site, you will be logged in until you close your browser.
+To log out, click on the :guilabel:`Log out` link in your documentation's :term:`flyout menu`.
+This is usually located in the bottom right or bottom left, depending on the theme design.
+This will log you out from the current domain,
+but not end any other session that you have active.
```

Comment:
This is not available anymore. Currently, there is no "Log out" button. We should probably remove this.